src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -144,6 +144,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             return;
         }
         _closing = true;
+
         // TODO:

```

Comment:
You could probably take out this whole todo block now too - looks like that's all been done now.